enterprise/server/util/cacheproxy/cacheproxy.go

```diff
@@ -319,8 +324,9 @@ func (c *CacheProxy) Read(req *dcpb.ReadRequest, stream dcpb.DistributedCache_Re
 	if resourceSize > 0 && resourceSize < bufSize {
 		bufSize = resourceSize
 	}
+	bufferedReader := bufio.NewReaderSize(reader, int(bufSize))
```

Comment:
I think this is a no-op since CopyBuffer is already using a buffer of size bufSize.

Reply:
I think you're right. I'd prefer to use bufio.NewReader but I think it would require pooling those Readers so i'm going to just leave this part as-is for now.